src/site/outline/outline.rst

```diff
@@ -177,41 +153,69 @@ assignment, to guard against the possibility of errors in recording marks. It is
+redeem 50% of their lost marks via *assignment recoupment*.
+
+Students will have **1 week** after their marked assignment is returned to incorporate feedback, correct mistakes, and
+resubmit their assignments for additional marking. All corrected mistakes can recoup 50% of the lost marks on that
+assignment. For example, if a student obtained 80/100 on the assignment after the regular submission, and the student
+correctly fixes errors that account for 12 of the 20 lost marks and resubmits for recoupment, the student will gain 6
+more marks for a total of 86/100 on the assignment. As always, no late submissions will be accepted and all work will be
+checked for plagiarism or cheating.
+
+There are some conditions, however. The marker will inform the student if their assignment is eligible for recoupment;
+only assignments deemed eligible may be resubmitted. Only assignments that were completed and attempted in earnest are
+eligible. Students with missing code/functions/portions of the assignment will not be eligible. If the student has
+portions of code that demonstrate a lack of a sincere attempt, the assignment will not be eligible for recoupment.
+
+Further, it is the responsibility of the student to make clear to the marker what exactly has been updated and changed.
+If the marker is unable to quickly determine what fixes the student has made, the resubmission will not be considered
+for recoupment. This could be in the form of notes in the Moodle submission with corresponding comments within the code.
+
 
 Tests
 =====
 
-* No cheating of any form is allowed
-* Tests and exams are closed book/laptop/notebook/etc.
-* Calculators will not be permitted
-* Use of a cell phone is not permitted
-* The possession of unauthorized devices during tests and examinations constitutes violation of academic integrity, as per section 3.8.2(b)(v) of the academic calendar
-* Anything that a reasonable person would deem as cheating is not permitted
+No cheating of any form is allowed. Tests and exams are closed book/laptop/notebook/etc. and calculators, cell phones
+and/or similar devices are not permitted. The possession of unauthorized devices during tests and examinations
+constitutes violation of academic integrity, as per section 3.8.2(b)(v) of the academic calendar. Further, anything that
+a reasonable person would deem as cheating is not permitted and will be investigated fully as academic misconduct.
+
+Test format will be in person; however, if necessary, the format *may* be changed to online. Students will be informed
+of the change as soon as possible.
 
 
+Missed Tests
+------------
+
+There are no make-up tests. If a student is unable to write a test, the weight of their test will be added to the
+subsequent test or final exam, whichever comes first. If both tests are missed, the weight of the tests will be added to
+the final exam.
```

Comment:
Does the part on _subsequent tests_ make sense the way it is written?

Reply:
I'd probably use the wording _following test_, but I think either is clear enough.

It may also be worth reiterating that there are two tests, or going into the specifics if it feels like it's not clear enough for you. Like, _miss test 1 => test 2 weight X% final Y%_ etc. It's a lot of text, so a table would probably be clearest.

Reply:
It's _probably_ fine without getting too explicit. I'll leave it the way it is, but if there is a lot of confusion, things will get updated.